_posts/EternalFootman/2016-02-03-chapter-5.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+author: EternalFootman
+title: "Kit's Chapter 5 Excercises"
+---
+<iframe src="https://trinket.io/embed/python/b895e1570f?start=result" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+<iframe src="https://trinket.io/embed/python/7c65457b74?start=result" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+The input aspect works, but I had the problem on both of the 'done' value, necessary to escape the loop, makes processing the numbers impossible. Is there a way to put each integer value into a list without using other values of x?
```

Comment:
Good question- the addition etc should probably only happen in the `else` statement, since you don't want it to happen if the input is "done".  We'll talk more in class.